Adding in method (#3023)
Codecademy/docs: content/java/concepts/calendar/terms/getAvailableCalendarTypes/getAvailableCalendarTypes.md

---
Title: '.getAvailableCalendarTypes()'

> Description: 'Returns an unmodifiable Set containing all calendar types supported by Calendar in the runtime environment'

Reviewer: ```suggestion
Description: 'Returns an unmodifiable Set containing all calendar types supported by Calendar in the runtime environment.'
```

Reviewer: This description is an exact copy of the oracle docs language, that is plagiarism. We do not permit plagiarism on docs. All content must be original. Revise the description and the text on line 15 with an explanation in your own words.

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The `getAvailableCalendarTypes()` method belongs to the `Calendar` class, which is included in the java.util package. This method returns an unmodified Set that contains all calendar types supported by `Calendar` in the Java runtime environment.

Reviewer: ```suggestion
The `.getAvailableCalendarTypes()` method belongs to the `Calendar` class, which is included in the `java.util` package. This method returns an unmodified `Set` that contains all calendar types supported by `Calendar` in the Java runtime environment.
```

> 
> ## Syntax
> ```pseudo
> getAvailableCalendarTypes ()
> ```
> > **Note:** The calendar types don't include aliases, such as "gregorian" for "gregory"
> 

Reviewer: ```suggestion
## Syntax

```pseudo
getAvailableCalendarTypes()
```

> **Note:** The calendar types don't include aliases, such as "gregorian" for "gregory"

```

Author: with line  24 do you want me to remove the note portion?

Reviewer: The note can stay

## Example

> 

Reviewer: There should be a brief introductory sentence after the example header

 ```java

>  import java.util.Calendar;  

Reviewer: The example should use 2 spaces instead of tabs for indentations

import java.util.Set;  
public class CalendargetavailablecalendartypeExample2 {  
    public static void main(String[] args) throws InterruptedException {  
        System.out.println(" All Available Calendar types: ");  
        Set set = Calendar.getAvailableCalendarTypes();  
        for(Object s:set){  
            System.out.println(s);  
        }                     
    }  
}  
```

> 

Reviewer: There should be a sentence such as: "This code returns the following output:" Between the code block and the output shell block.
